src/www/javascript/opnsense_legacy.js

```diff
@@ -257,5 +261,6 @@ function hook_firewall_categories() {
             $(".opnsense-rules").change();
         });
         cat_select.change();
+        $('[data-toggle="tooltip"]').tooltip();
```

Comment:
I'm not sure if it matters, but in case existing selectors don't like being triggered twice, you can always add a unique class to the items added and only trigger those,